.github/workflows/tests.yml

```diff
@@ -43,54 +43,33 @@ jobs:
     container:
       image: "docker.io/library/fedora:${{ matrix.fedora_release }}"
     steps:
-      - name: Prepare container
+      - name: "📥 Prepare container"
         run: |
           echo "fastestmirror=1" >> /etc/dnf/dnf.conf
           echo "install_weak_deps=0" >> /etc/dnf/dnf.conf
           rm -fv /etc/yum.repos.d/fedora*modular*
           dnf -y upgrade
-          dnf -y install dnf-plugins-core findutils rpm-build rpmdevtools
+          dnf -y install dnf-plugins-core findutils git make rpm-build rpmdevtools
 
-      - name: Fetch .spec file
-        run: |
-          mkdir spec
-          # Fetch the spec file at the given commit, and prepend the commit sha to it
-          curl "https://raw.githubusercontent.com/osbuild/osbuild-composer/${GITHUB_SHA}/golang-github-osbuild-composer.spec" -o golang-github-osbuild-composer.spec
-          echo "%global commit ${GITHUB_SHA}" | cat - golang-github-osbuild-composer.spec > spec/golang-github-osbuild-composer.spec
-
-      - name: Install RPM build dependencies
-        run: dnf -y builddep spec/golang-github-osbuild-composer.spec
+      - name: "🗄️ Clone the repository"
+        uses: actions/checkout@v2
 
-      - name: Fetch sources
-        run: |
-          mkdir sources
-          spectool -g spec/golang-github-osbuild-composer.spec -C sources/
+      - name: "🛒 Install RPM build dependencies"
+        run: dnf -y builddep golang-github-osbuild-composer.spec
 
-      - name: Build SRPM
-        run: |
-          mkdir srpms
-          rpmbuild -bs \
-            --define "_sourcedir sources" \
-            --define "_srcrpmdir srpms" \
-            spec/golang-github-osbuild-composer.spec
+      - name: "🛠️ Build RPMs"
+        run: make rpm
 
-      - name: Build RPMs
+      - name: "🚚 Prepare RPMs for artifact upload"
         run: |
-          mkdir rpmbuild rpms build
-          rpmbuild -bb \
-            --define "_sourcedir `pwd`/sources" \
-            --define "_specdir `pwd`/spec" \
-            --define "_builddir `pwd`/rpmbuild" \
-            --define "_srcrpmdir `pwd`/srpms" \
-            --define "_rpmdir `pwd`/rpms" \
-            --define "_buildrootdir `pwd`/build" \
-            spec/golang-github-osbuild-composer.spec
+          mkdir rpms
+          cp -av rpmbuild/RPMS/*/*.rpm rpms/
```

Comment:
Please also copy the `rpmbuild/SRPMS/*.src.rpm` to get the srpm as an artifact as well.

Reply:
Oh, good idea.